deploy/ocm-agent-operator-managednotifications/10-managednotifications-cr.yaml

```
@@ -12,4 +12,28 @@ spec:
       resolvedBody: |-
         Your cluster's ElasticSearch deployment is detected as being at safe disk consumption levels and no additional action on this issue is required.
       severity: Info
-      summary: "ElasticSearch reaching disk capacity" 
\ No newline at end of file
+      summary: "ElasticSearch reaching disk capacity" 
+    - activeBody: |-
+        Your cluster requires you to take action as its ElasticSearch cluster has hit the high watermark due to low free disk space. Some shards will be re-allocated to different nodes if possible.  Please consult the following documentation for reference: https://docs.openshift.com/container-platform/latest/logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.html#Elasticsearch-Node-Disk-High-Watermark-Reached
+      name: ElasticsearchNodeDiskWatermarkReached
+      resendWait: 24
+      resolvedBody: |-
+        Your cluster has no more ElasticsearchNodeDiskWatermarkReached alert and no additional action on this issue is required.
+      severity: Info
+      summary: "ElasticSearch Disk High Watermark Reached" 
+    - activeBody: |-
+        Your cluster requires you to take action as its ElasticSearch cluster is predicted to be out of disk space within the next 6 hours based on current disk usage. Please consult the following documentation for reference: https://docs.openshift.com/container-platform/latest/logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.html#Elasticsearch-Disk-Space-is-Running-Low
+      name: ElasticsearchDiskSpaceRunningLow
+      resendWait: 24
+      resolvedBody: |-
+        Your cluster has no more ElasticsearchDiskSpaceRunningLow alert and no additional action on this issue is required.
+      severity: Info
+      summary: "ElasticSearch Cluster low on disk space" 
+    - activeBody: |-
+        Your cluster requires you to take action as its ElasticSearch cluster health status has been RED for at least couple of minutes. The cluster does not accept writes, shards may be missing, or the master node has not been elected yet. Please consult the following documentation for reference: https://docs.openshift.com/container-platform/4.10/logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.html#Elasticsearch-Cluster-Health-is-Red
```

And here; also I would get rid of the "at least a couple of minutes" bit.

```suggestion
        Your cluster requires you to take action as its ElasticSearch cluster health status has been detected as RED. The cluster does not accept writes, shards may be missing, or the master node has not been elected yet. Please consult the following documentation for reference: https://docs.openshift.com/container-platform/4.10/logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.html#elasticsearch-cluster-health-is-red
```